rules/jsll/package.json

```diff
@@ -47,29 +48,31 @@
   ],
   "homepage": "https://sonarwhal.com",
   "keywords": [
-    "jsll",
-    "rule",
     "sonarwhal",
-    "sonarwhal-rule"
+    "sonarwhal-rule",
+    "jsll",
+    "jsll-rule"
   ],
   "license": "Apache-2.0",
   "main": "dist/src/index.js",
   "name": "sonarwhal-rule-jsll",
   "peerDependencies": {
-    "sonarwhal": "^0.25.0"
+    "sonarwhal": "^0.28.2"
   },
   "scripts": {
-    "ava": "ava",
     "build": "npm run clean && npm-run-all build:*",
+    "build-release": "npm run clean && npm run build:assets && tsc --inlineSourceMap false --removeComments true",
     "build:assets": "cpx \"./{src,tests}/**/{!(*.ts),.!(ts)}\" dist",
-    "build:ts": "tsc --outDir \"./dist\"",
+    "build:ts": "tsc",
     "clean": "rimraf dist",
-    "test": "npm run lint && npm run build && ava",
     "lint": "npm-run-all lint:*",
-    "lint:js": "eslint --ext md --ext ts --ignore-pattern dist .",
-    "lint:md": "markdownlint README.md",
+    "lint:js": "eslint . --cache --ext js --ext md --ext ts --ignore-path .eslintignore --report-unused-disable-directives",
+    "lint:md": "markdownlint --ignore CHANGELOG.md *.md",
+    "test": "npm run lint && npm run build && nyc ava",
+    "watch": "npm run build && npm-run-all --parallel -c watch:*",
+    "watch:assets": "npm run build:assets -- -w --no-initial",
+    "watch:test": "ava --watch",
     "watch:ts": "npm run build:ts -- --watch",
-    "init": "npm install && npm run build",
     "sonarwhal": "node node_modules/sonarwhal/dist/src/bin/sonarwhal.js"
   },
   "repository": "Microsoft/sonarwhal-microsoft-rules",
```

Comment:
Can you also update the repo to `sonarwhal-rules-microsoft`

Reply:
@molant I think this should be `sonarwhal-rules-microsoft-jssll` and chisel to `sonarwhal-rules-microsoft-chisel`, if not, I think sonarwhal are not going to be able to load them.

Reply:
That's the name of the repository which currently is `Microsoft/sonarwhal-rules-microsoft`, otherwise the link from npm doesn't work 😛

Reply:
yep, I need to read more hahahaha